### DISCUSSION TITLE
v-model binding as number

### DISCUSSION THREAD

**Discussion by FabianGML** [dm4t2/vue-currency-input#422] [Q&A]
If I bind an input to a ref and that ref is a string, is there a way to ensure the input value is a number? I know I can use Number() to transform the value, but I need the ref to never be a string at all

**Comment by dm4t2** [comment]
The value is always emitted as a number if you use the default "auto emit" configuration: https://dm4t2.github.io/vue-currency-input/guide.html#auto-emit

**Comment by FabianGML** [comment]
Yes! I'm Sorry, it was my fault.

However, I have another question.
How can I format the input before the user focuses on it? When the component is rendered, the number appears without any formatting, but in the playground, the input shows the EUR prefix and everything is formatted. I've been trying to figure out how to achieve this, but I haven't found a way.
